SPEC.md

```diff
@@ -2080,16 +2094,14 @@ Example input:
       "id": "mouse_size",
       "variables": ["name", "height"],
       "data": {
-        "name": "Pinky",
-        "height": 7
+        "name": "Pinky"
       }
     },
     {
       "id": "pig_weight",
       "variables": ["name", "weight"],
       "data": {
-        "name": "Porky",
-        "weight": 1000
+        "name": "Porky"
```

Comment:
`variables` no longer holds the map keys; we should either cut the extra `variables` entries or change these numbers into quoted strings instead of removing them.

Reply:
Fixed